code/pyth/client/client.en.ts

```diff
@@ -0,0 +1,19 @@
+import * as web3 from "@solana/web3.js"
+import * as pyth from "@pythnetwork/client"
+
+const connection = new web3.Connection(
```

Comment:
Could you wrap the code like the rest of the js code so that it is auto runnable? 

Just to follow the same standard throughout
```javascript
(async () => {
  
})();
```

Reply:
Got it @jacobcreech really sorry for this mistakes